### DISCUSSION TITLE
Material database

### DISCUSSION THREAD

**Discussion by ManuelRom1** [arkypita/LaserGRBL#1833] [Q&A] [answered]
Is it possible to export and import the material database? Where is the file located?

Thanks for your attention

**Comment by arkypita** [comment, marked as answer]
C:\Users\<yourname>\AppData\Roaming\LaserGRBL\UserMaterials.psh

It's XML

**Reply by ManuelRom1** [reply]
Thank you very much, I edited and translated the entire database by hand and when I changed PCs I didn't know how to copy it.
Knowing the location and format is already much easier.
Thanks again
